ios/RNMagicScript/components/UiNodes/UiSliderNode.swift

```diff
@@ -141,31 +144,40 @@ import SceneKit
             self.height = height
         }
 
-        if let min = Convert.toCGFloat(props["min"]) {
+        if let max = Convert.toCGFloat(props["max"]),
+            let min = Convert.toCGFloat(props["min"]) {
+            updateMinMax(min: min, max: max)
+        } else if let max = Convert.toCGFloat(props["max"]) {
+            self.max = max
+        } else if let min = Convert.toCGFloat(props["min"]) {
             self.min = min
         }
 
-        if let minLabel = Convert.toString(props["minLabel"]) {
-            self.minLabel = minLabel
+        if let value = Convert.toCGFloat(props["value"]) {
+            self.value = value
         }
 
-        if let max = Convert.toCGFloat(props["max"]) {
-            self.max = max
+        if let minLabel = Convert.toString(props["minLabel"]) {
+            self.minLabel = minLabel
         }
 
         if let maxLabel = Convert.toString(props["maxLabel"]) {
             self.maxLabel = maxLabel
         }
 
-        if let value = Convert.toCGFloat(props["value"]) {
-            self.value = value
-        }
-
         if let foregroundColor = Convert.toColor(props["foregroundColor"]) {
             self.foregroundColor = foregroundColor
         }
     }
 
+    private func updateMinMax(min: CGFloat, max: CGFloat) {
+        if (min < max) && (_min != min || _max != max) {
```

Comment:
Should we use `<` or `<=`? What if min == max?

Reply:
According to discussion and Lumin behaviour there is no case to support min == max.